scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -748,15 +748,15 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-secondary">Pa
               <td width="5%">#{{loop.index}}</td>
               {% if variant.category == 'snv' %}
                 <td width="25%"><a href="{{ url_for('variant.variant', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" target="_blank"><strong>{{variant.display_name[:30]}}</strong></a></td>
-              {% else %}
+                <td width="25%">
+    	             {% for gene in variant.genes %}
+                    <a href="http://www.genenames.org/cgi-bin/gene_symbol_report?match={{ gene.hgnc_symbol }}" target="_blank">{{ gene.hgnc_symbol }}&nbsp;<span class="badge badge-secondary">{{gene.region_annotation}}</span></a><br>
+                   {% endfor %}
+                </td>
+              {% else %} <!-- SVs -->
```

Comment:
How about other categories, like cancer and str? Cancer will go nicely with snv (and cancer_sv obviously with sv). Str could use a gene or str_repid if none is set (older cases).

Reply:
Cancer SVs and SNVs will look like RD SVs and SNVs. I'll fix strs as you suggested!